AGENTS.md

```
@@ -0,0 +1,81 @@
+style: Updates layout colors.
+
+The colors are now more vibrant and the layout is more modern.
+```
+
+```
+feat(deps): Upgrades `foo` dependency.
+
+BREAKING CHANGE: the bump of the dependency is not compatible with the previous version.
+```
+
+## Commiting Changes
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)